conf/messages

```diff
@@ -1325,6 +1325,7 @@ users.user.details.status=Status
 
 users.user.details.role.hint.classification=Manage ATaR, liability, correspondence and miscellaneous cases.
 users.user.details.role.hint.gateway=Assign ATaR cases. Create and assign Liability, Correspondence and Miscellaneous cases.
+users.user.details.role.hint.not.found=Role or isGateway not found
```

Comment:
will probably need better and new content for this message and welsh yeah if it ever gets decided we need to handle these match cases or not